startup-prod.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+BASH_ENV=.env_app
```

Comment:
Is this actually working? It stopped working for me for some reason.

Reply:
I guess there was an extra space after the `APP_ROOT` variable, which I have fixed:
`APP_ROOT= `-> `APP_ROOT=`